### PR TITLE
Add Watts unit to nv_gpu_power_usage description

### PR DESCRIPTION
**Pull request by rmccorm4**
Updating docs to mention power unit, matching help description:
```
# HELP nv_gpu_power_usage GPU power usage in watts
```

### CODE REVIEW

**Review by dyastremsky — COMMENTED** [review]
Great clarification!

Nit: watts should not be capitalized unless using the symbol "W". I'd approve, but it looks like we don't capitalize other units, so I'd rather keep this consistent if you don't mind.

File: docs/user_guide/metrics.md [hunk 1]
```diff
@@ -193,8 +193,8 @@ If building Triton locally, the `TRITON_ENABLE_METRICS_GPU` CMake build flag can
 
 |Category      |Metric          |Metric Name |Description                            |Granularity|Frequency    |
 |--------------|----------------|------------|---------------------------|-----------|-------------|
-|GPU Utilization |Power Usage   |`nv_gpu_power_usage` |GPU instantaneous power                |Per GPU    |Per interval   |
-|              |Power Limit     |`nv_gpu_power_limit` |Maximum GPU power limit                |Per GPU    |Per interval   |
+|GPU Utilization |Power Usage   |`nv_gpu_power_usage` |GPU instantaneous power, in Watts      |Per GPU    |Per interval   |
+|              |Power Limit     |`nv_gpu_power_limit` |Maximum GPU power limit, in Watts      |Per GPU    |Per interval   |
 |              |Energy Consumption|`nv_energy_consumption` |GPU energy consumption in joules since Triton started|Per GPU|Per interval|
```
Comment (dyastremsky) [inline docs/user_guide/metrics.md:198]: Also, if you have the chance, would you mind adding a comma here before "in joules"? I don't care one way or the other about the comma, but would like to be consistent within the table/document.

Note: not required for approval, as this is outside the scope of your change.

**Review by fpetrini15 — APPROVED** [review]

File: docs/user_guide/metrics.md (hunk 1 above)
Comment (rmccorm4) [reply docs/user_guide/metrics.md]: Added comma, fixed capitalization, re-aligned GPU metric table. I know not all tables are aligned, but some have quite long descriptions that it's somewhat moot for those anyway.
image

**Review by dyastremsky — APPROVED** [review]
Great work on these fantastic changes, Ryan!
